test/software/amazon/ion/EquivTimelineTest.java

```diff
@@ -51,4 +53,19 @@ protected void checkEquivalence(IonValue left, IonValue right,
         assertEquals("calendar", lTime.calendarValue(), rTime.calendarValue());
         assertEquals("date", lTime.dateValue(), rTime.dateValue());
     }
+
+    @Override
+    public void roundTripEquivalence(IonDatagram input, boolean myExpectedEquality) throws IOException {
+        IonDatagram[] data = roundTripDatagram(input);
+        for(int i = 0; i < data.length - 1; i++){
+            runEquivalenceChecks(data[i], myExpectedEquality);
+            for(int j = i + 1; j < data.length; j++) {
+                for(int sexpIndice = 0; sexpIndice < data[i].size(); sexpIndice++) {
```

Comment:
`Indice` -> `Index` ?

Reply:
equivalent.

Reply:
I have never seen a variable named `Indice`. Dictionary.com doesn't think it's a word, while thefreedictionary.com says it's a synonym for "index". To each his own, I suppose.

Reply:
Some perceive a small shading of difference between simple numerical-offset array indices and the more sophisticated data structures of database indexes or full-text indexes. ("The indexes were corrupted because the update code has an off-by-one error in its indices.")

Reply:
I'm not denying that "indices" is a word. I don't care whether "index" is pluralized as "indices" or "indexes". All I'm saying is that I've never seen "indice" in singular form, and I found it distracting until I found out that 50% of the online dictionaries I consulted say it's an uncommonly-used synonym for "index". These comments are not blockers for this PR.